index.html

```diff
@@ -116,7 +159,7 @@ <h1>Tonic</h1>
                  <p class="project_description">
                   A daily selection of privately personalized reads; no accounts or sign-ups required.
                  </p>
-            </div>
+            
```

Comment:
- [ ]  Kyndly stick to the figma design as much as posible, in the cards section you added too much margin to the left and rigt side of the cards in comparison to the figma design.

## Figma design 

![image](https://user-images.githubusercontent.com/95243996/201197659-279838ca-55cb-4b8b-949f-423e2aecad5b.png)

## Your design

![image](https://user-images.githubusercontent.com/95243996/201197817-e5919336-d89e-4470-bc9d-03d3970dd2d3.png)